package.json

```diff
@@ -43,44 +43,50 @@
   },
   "homepage": "https://github.com/Microsoft/vso-team-calendar#readme",
   "scripts": {
-    "postinstall": "cd legacy && npm install && cd ..",
-    "clean": "rimraf ./dist && cd legacy && npm run clean && cd ..",
-    "compile": "npm run clean && webpack --mode production",
-    "compile:dev": "npm run clean && webpack --mode development",
-    "build": "npm run compile && cd legacy && npm run build && cd ..",
+    "clean": "rimraf ./dist",
+    "compile": "npm run clean && webpack --mode production --config ./webpack.config.js",
+    "compile:dev": "npm run clean && webpack --mode development --config ./webpack.config.js",
+    "build:release": "npm run compile",
     "build:dev": "npm run compile:dev && npm run postbuild",
-    "postbuild": "npm run package-extension -- --rev-version",
+    "postbuild": "npm run package:dev -- --rev-version",
+    "package:dev": "node ./scripts/packageDev",
+    "package:release": "node ./scripts/packageRelease",
     "package-extension": "tfx extension create --manifest-globs azure-devops-extension.json src/*.json",
-    "publish-extension": "tfx extension publish --manifest-globs azure-devops-extension.json src/*.json"
+    "publish-extension": "tfx extension publish --manifest-globs azure-devops-extension.json src/*.json",
+    "dev": "webpack-dev-server --hot --progress --static ./ --https --port 8888"
```

Comment:
Check this command , think it has older command arguments that are not supported by the latest webpack.

Reply:
npm run build works as expected, added few changes